java/flight/flight-core/src/main/java/org/apache/arrow/flight/FlightEndpoint.java

```diff
@@ -54,13 +56,27 @@ public FlightEndpoint(Ticket ticket, Location... locations) {
    * Constructs a new endpoint with an expiration time.
    *
    * @param ticket A ticket that describe the key of a data stream.
+   * @param expirationTime (optional) When this endpoint expires.
    * @param locations  The possible locations the stream can be retrieved from.
    */
   public FlightEndpoint(Ticket ticket, Instant expirationTime, Location... locations) {
+    this(ticket, expirationTime, "", locations);
```

Comment:
It seems like the default value is `""` in the C implementation. Should I stick with that here?

Reply:
Use `null`